pulser-simulation/pulser_simulation/simulation.py

```diff
@@ -1150,6 +1188,14 @@ def __init__(
         with_modulation: bool = False,
     ) -> None:
         """Instantiates a Simulation object."""
+        with warnings.catch_warnings():
+            warnings.simplefilter("always")
+            warnings.warn(
+                DeprecationWarning(
+                    "The `Simulation` class is deprecated,"
+                    " use `QutipEmulator.from_sequence` instead."
+                )
+            )
```

Comment:
Should we delete `Simulation.draw()` now?

Reply:
I am not sure, as we have `draw_sequence` and `draw_samples`, it thought it was not absurd to keep a different draw for `Simulation` and `QutipEmulator` (there is one more option `interp_pts`)

Reply:
Yeah, I guess you're right, we can just leave it there and it will go away once `Simulation` is removed